package.json

```diff
@@ -4,7 +4,7 @@
   "description": "JS API that serves food and meal data.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha test/*"
```

Comment:
Awesome work getting this set up. This will be great for keeping eyes on the test suite throughout the duration of the project.